tests/IntegrationTests/WebAppUiTests/UiTestHelpers.cs

```diff
@@ -66,5 +125,129 @@ await page.Context.Tracing.StopAsync(new()
                 Path = "PlaywrightTrace.zip"
             });
         }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="testAssemblyLocation">The path to the test's directory</param>
+        /// <param name="appLocation">The path to the processes directory</param>
+        /// <param name="executableName">The name of the executable that launches the process</param>
+        /// <param name="portNumber">The port for the process to listen on</param>
+        /// <returns></returns>
+        public static Process? StartProcessLocally(string testAssemblyLocation, string appLocation, string executableName, string? portNumber = null)
```

Comment:
shouldn't port number be an int/uint? then stringified in the method?

Reply:
it eliminates a class of possible misuse. If the parameter is supposed to always be a port number, then what happens if something other than a port number is passed in? If it can be something other than a port number, then it needs to not be called `portNumber`